Changes

```diff
@@ -92,6 +92,9 @@ Working version
 - #8716: Optimize [Array.fill] and [Hashtbl.clear] with a new runtime primitive
   (Alain Frisch, review by David Allsopp, Stephen Dolan and Damien Doligez)
 
+- #8832: List.find_some : ('a -> 'b option) -> 'a list -> 'b option
```

Comment:
This needs to be changed to `find_map` (as well as the title of the PR please).

Reply:
Fixed, thanks.